airbyte-integrations/bases/base-normalization/setup.py

```diff
@@ -32,7 +32,14 @@
     author_email="contact@airbyte.io",
     url="https://github.com/airbytehq/airbyte",
     packages=setuptools.find_packages(),
-    install_requires=["airbyte-protocol", "pyyaml", "jinja2"],
+    install_requires=[
+        "airbyte-protocol",
+        "pyyaml",
+        "jinja2",
+        # TODO: change to dbt-mysql after this PR is merged:
+        # https://github.com/dbeatty10/dbt-mysql/pull/57
+        "dbt-mysql @ git+https://github.com/airbytehq/dbt-mysql-1@patch-1#egg=dbt-mysql",
+    ],
```

Comment:
> I'm unable to run the tests locally either...
> 
> Have you added the MySQL-adapter git repo to `airbyte-integrations/bases/base-normalization/dbt-project-template/packages.yml` btw?

@ChristopheDuong, I did not add it to the `packages.yml` but added it here. It was necessary because when I started working on this PR, there was a bug in the `dbt-mysql` release, and a [patch](https://github.com/dbeatty10/dbt-mysql/pull/57) was needed. This patch just got merged 2 days ago. I will test and see if I can just add it to the dbt dependency instead of here.

Reply:
Although the patch has been merged, however, we cannot add `dbt-mysql` under `packages.yml` because it has no `dbt_project.yml`:

```
no dbt_project.yml found
```

So we can only specify it in `setup.py`.